Translated/Dialog/Adila/ssn_012030.csv

```diff
@@ -7,7 +7,7 @@ ssn_012030_0103#0,"""If you don't, I'll be happy to explain it to you!"""
 ssn_012030_0104#0,"""Alright! Then listen up!"""
 ssn_012030_0105#0,"""Basically, Special Abilities are little bonuses that tweak the performance of your weapons and armor."""
 ssn_012030_0106#0,"""Next time you've got your equipment data in front of you, take a careful look at it."""
-ssn_012030_0107#0,"""So... You can put multiple Special Abilities on the same piece of equipment, and even replace them if you're tired of them."""
-ssn_012030_0108#0,"""You can add Special Abilities down at the Item Lab. But you'll need Special Ability Capsules if you want to try it yourself..."""
+ssn_012030_0107#0,"""So... you can mix Special Abilities with different equipment, or even match Special Abilities across all of them."""
```

Comment:
So... You can put multiple different Special Abilities on the same piece of equipment, and even replace them if you're tired of them.
-a fix to this would probably make the most sense.
-fix also needs to be duplicated to ssn_012031.csv.

Reply:
honestly dunno what i was thinking here. anyways, fixed and amended to 012031